q2_qemistree/_plot.py

```diff
@@ -96,10 +96,51 @@ def format_labels(feature_metadata, category, ms2_label, parent_mz):
     return '\n'.join(labels)
 
 
-def plot(output_dir: str, table: biom.Table, tree: NewickFormat,
-         feature_metadata: pd.DataFrame, category: str,
-         color_palette: str = 'Dark2', ms2_label: bool = False,
-         parent_mz: str = None) -> None:
+def format_barplots(table: biom.Table):
+    barplots = []
+    barplots.append('DATASET_MULTIBAR')
+    barplots.append('SEPARATOR TAB')
+    barplots.append('DATASET_LABEL\tRelative Abundance')
+    barplots.append('COLOR\t#ff0000')
+
+    table = table.norm(axis='observation', inplace=False)
+    table = pd.DataFrame(table.matrix_data.toarray(), table.ids('observation'),
+                         table.ids('sample'))
+
+    field_labels = list(table.columns)
+    field_colors = values_to_colors(field_labels, 'husl').values()
+
+    barplots.append('FIELD_COLORS\t'+('\t').join(field_colors))
```

Comment:
The parentheses around `'\t'` are not necessary.

Reply:
That's right, removed!